cmd/sops/encrypt.go

```diff
@@ -64,8 +64,10 @@ func encrypt(opts encryptOpts) (encryptedFile []byte, err error) {
 	if err != nil {
 		return nil, common.NewExitError(fmt.Sprintf("Error unmarshalling file: %s", err), codes.CouldNotReadInputFile)
 	}
-	if err := ensureNoMetadata(opts, branches[0]); err != nil {
-		return nil, common.NewExitError(err, codes.FileAlreadyEncrypted)
+	if len(branches) != 0 {
```

Comment:
In which case can `len(branches)` be negative? From https://go.dev/ref/spec#Length_and_capacity I cannot see why it could be negative, and I couldn't find anything online either.

Reply:
Technically, none. Just more defensive.